### PR TITLE
Cleanup: Handbook chapters

### PR DESCRIPTION
**Pull request by MarkusAbtion**
We have some loose files in the root of this repository. A few of them have to do with the Handbook so I've moved them into their own folder. 

This will hopefully give a quicker overview of the `guidelines` and will allow us to present the Handbook and its contents in the corresponding README file.

Screenshot 2021-02-11 at 09 15 16

### CODE REVIEW

**Comment by prrrnd** [comment]
❤️

**Comment by mathias-abtion** [comment]
Nice!

**Review by mathias-abtion — APPROVED** [review]

File: handbook_wip/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Handbook WIP
```
Comment (hcarreras) [inline handbook_wip/README.md:1]: I'm afraid that we need to maintain 2 READMEs now when we change a file

**Review by hcarreras — COMMENTED** [review]
Super cool. 
I'm not convinced of the double README until we have that automated.

File: handbook_wip/README.md (hunk 1 above)
Comment (MarkusAbtion) [reply handbook_wip/README.md:1]: I think it's a nice way to introduce the handbook as a concept. I don't consider it a README we'll need to update regularly.

File: README.md [hunk 2]
```diff
@@ -19,16 +19,16 @@
...
 - [WordPress setup](./wordpress/)
-- Handbook_WIP
-  - [How to facilitate team meetings](./how_to_facilitate_team_meetings.md)
-  - [Procedure for traffic](./procedure_for_traffic.md)
-  - [How and why we do design research](./how_and_why_we_do_design_research.md)
-  - [Client discontinue hosting or SLA](./client_discontinue_hosting_or_SLA.md)
-  - [TDD Testdriven development](./tdd_testdriven_development.md)
-  - [Invoicing_guidelines](./invoicing_guidelines.md)
-  - [Kick-off meeting RoR](./kick_off_meeting_RubyOnRails.md)
+- [Handbook_WIP](./handbook_wip)
```
[the remark this answers is not in the document]
Comment (MarkusAbtion) [reply README.md:22]: Ah ok. I think we should update that then :)

File: handbook_wip/README.md (hunk 1 above)
Comment (hcarreras) [reply handbook_wip/README.md:1]: We are able to see the same structure just above the README. 
I will delete this and get it merged. We can talk about it outside the PR.

**Comment by hcarreras** [comment]
I'm merging this now as it is fairly small. Let's keep improving the structure of the guidelines 💪

**Pull request merged**